doc/api/test.md

```diff
@@ -316,6 +316,29 @@ Otherwise, the test is considered to be a failure. Test files must be
 executable by Node.js, but are not required to use the `node:test` module
 internally.
 
+## `runFiles([options])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `options` {Object} Configuration options for running test files. The following
+  properties are supported:
+  * `concurrency` {number|boolean} If a number is provided,
+    then that many files would run in parallel.
+    If truthy, it would run (number of cpu cores - 1)
+    files in parallel.
+    If falsy, it would only run one file at a time.
+    If unspecified, subtests inherit this value from their parent.
+    **Default:** `true`.
+  * `files`: {Array} An array containing the list of files to run.
+    **Default** matching files from [test runner execution model][].
+  * `signal` {AbortSignal} Allows aborting an in-progress test file.
+  * `timeout` {number} A number of milliseconds the test file will fail after.
+    If unspecified, subtests inherit this value from their parent.
+    **Default:** `Infinity`.
+* Returns: {Promise} Resolved with `undefined` once all test files complete.
```

Comment:
Would be neat if it fulfilled with the results and not `undefined` (or at leas the tap output string?)

Reply:
Would it reject if the tests fail?

Reply:
@benjamingr what is the result of a test?

Reply:
Test name + pass/fail?

Reply:
well pass/fail is weather the promise is rejected or resolved, there is no test name since this promise is for all tests